frontend/public/components/edit-yaml.jsx

```diff
@@ -79,22 +101,28 @@ export const EditYAML_ = connect(stateToProps)(
           this.onCancel = 'onCancel' in props ? props.onCancel : history.goBack;
           this.updateYAML = this.updateYAML.bind(this);
           this.loadCSVs = this.loadCSVs.bind(this);
-
+          this.showResults = this.showResults.bind(this);
+          this.onRetry = this.onRetry.bind(this);
           if (this.props.error) {
             this.handleError(this.props.error);
           }
         }
 
-        getModel(obj) {
-          if (_.isEmpty(obj)) {
-            return null;
-          }
-          const { models } = this.props;
-          return models.get(referenceFor(obj)) || models.get(obj.kind);
+        handleError(error) {
+          this.setState({ errors: _.isEmpty(this.props.error) ? null : [error], success: null });
         }
 
-        handleError(error) {
-          this.setState({ error, success: null });
+        handleErrors(resourceObject, error) {
+          const resourceName = resourceObject?.metadata?.name;
+          this.setState((prevState) => {
+            const errors = [
+              ...(prevState.errors || []),
+              resourceName ? `${resourceName}: ${error}` : error,
```

Comment:
Do we need kind+name here since resource name is not unique? Or is this normally in the error message (in which case, maybe we don't need the prefix)?

Reply:
The error references the part of the yaml that has the issue, so if the ns is missing, the ns is called out, but not the kind. I'll add the kind here as well.
kind name: error ie 
Secret example 1: namespace "joesnamespace" doesn't exist